rm/src/lib.rs

```diff
@@ -84,16 +86,31 @@ pub fn run(options: Options, ctx: impl term::Context) -> anyhow::Result<()> {
         .join("namespaces")
         .join(options.urn.encode_id());
 
-    if !options.prompt
-        || term::confirm(format!(
+    if options.prompt
+        && !term::confirm(format!(
             "Are you sure you would like to delete {}?",
             term::format::dim(namespace.display())
         ))
     {
-        rad_untrack::execute(&options.urn, rad_untrack::Options { peer: None }, &profile)?;
-        fs::remove_dir_all(namespace)?;
-        term::success!("Successfully removed project {}", options.urn);
+        return Ok(());
     }
 
+    // A project is only considered to exist if it is tracked one or more times.
+    tracking::untrack_all(
```

Comment:
Mmh, is this equivalent to calling `rad_untrack::execute` with `peer == None`?

Reply:
Is there something wrong with the implementation?  This is part of the intent of the PR.

Reply:
As far as I understand the comments in https://github.com/radicle-dev/radicle-cli/issues/189, this PR should only mostly remove the `--all` option from `rad-untrack`. And thus wondering, what the reason to replace `rad_untrack::execute()` with `tracking::untrack_all()` was. Just don't know if is supposed to behave the same or if's different for the very same reason that I don't fully get yet :wink:

Reply:
It should be ;)  calling rad_untrack::execute with peer = None is the internal equivalent of 'rad untrack --all'.  I was confused whether a problem was spotted or if it was curiosity.

Check the changes at https://github.com/radicle-dev/radicle-cli/pull/222/files#diff-48f638317ddb6a247ea94a4bca3e12f78ade01dc2233a6e818e2fef066abd524L109-L116

Reply:
Thanks for the explanation!